cockatrice/src/gamesmodel.cpp

```diff
@@ -23,25 +24,37 @@ enum GameListColumn
     SPECTATORS
 };
 
-const QString GamesModel::getGameCreatedString(const int secs) const
+const QString GamesModel::getGameCreatedString(const int secs)
 {
+    static const QTime zeroTime{0, 0};
+    static const int halfHourSecs = zeroTime.secsTo(QTime(1, 0)) / 2;
+    static const int halfMinSecs = zeroTime.secsTo(QTime(0, 1)) / 2;
+    static const int wrapSeconds = zeroTime.secsTo(zeroTime.addSecs(-halfHourSecs)); // round up
 
-    QString ret;
-    if (secs < SECS_PER_MIN * 2) // for first min we display "New"
-        ret = tr("New");
-    else if (secs < SECS_PER_MIN * 10) // from 2 - 10 mins we show the mins
-        ret = QString("%1 min").arg(QString::number(secs / SECS_PER_MIN));
-    else if (secs < SECS_PER_MIN * 60) { // from 10 mins to 1h we aggregate every 10 mins
-        int unitOfTen = secs / SECS_PER_TEN_MIN;
-        QString str = "%1%2";
-        ret = str.arg(QString::number(unitOfTen), "0+ min");
-    } else { // from 1 hr onward we show hrs
-        int hours = secs / SECS_PER_HOUR;
-        if (secs % SECS_PER_HOUR >= SECS_PER_MIN * 30) // if the room is open for 1hr 30 mins, we round to 2hrs
-            ++hours;
-        ret = QString("%1+ h").arg(QString::number(hours));
+    if (secs >= wrapSeconds) { // QTime wraps after a day
+        return tr(">day");
     }
-    return ret;
+
+    QTime total = zeroTime.addSecs(secs);
+    QTime totalRounded = total.addSecs(halfMinSecs); // round up
+    QString form;
+    int amount;
+    if (totalRounded.hour()) {
+        amount = total.addSecs(halfHourSecs).hour(); // round up separately
+        form = tr("%1%2 h", "short age in hours", amount);
```

Comment:
Would like this to be hr, to be in alignment with "min"
```suggestion
        form = tr("%1%2 hr", "short age in hours", amount);
```

Reply:
as mentioned by @tooomm the appropriate unit shorthand is h https://en.wikipedia.org/wiki/Non-SI_units_mentioned_in_the_SI

Reply:
if you want it to be hr, you should add that to the english translation file instead: https://github.com/Cockatrice/Cockatrice/issues/4003